README.md

```diff
@@ -125,6 +113,40 @@ Repo of the Python module JIWER: https://pypi.org/project/jiwer/
 It computes the minimum-edit distance between the ground-truth sentence and the hypothesis sentence of a speech-to-text API.
 The minimum-edit distance is calculated using the python C module python-Levenshtein.
 
+## Execution
+
+```
+python analyze.py --config_file config.ini --log_level DEBUG
+```
+
+# Experimenting
+Use the `experiment.py` script to execute a series of Transcription/Analyze experiments to optimize SpeechToText parameters. 
+
+## Setup
+
```

Comment:
Would it make any sense to get the log level from the config file instead of a script execution flag?

Reply:
I thought about doing that but landed on the side of cmdline bc changing the log level is something that I'd want to do at runtime without having to go edit any other files. I can toggle it in the terminal which is more convenient if I'm debugging. I can be swayed though, either way would work.

Reply:
for debugging, I agree with @gecock : I would rather do it at the cmdline as needed instead of changing the config file.

Reply:
@tnixa Please confirm this is resolved to your satisfaction and resolve conversation